main.css

```diff
@@ -0,0 +1,61 @@
+header {
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+  padding: 25px;
+  align-items: center;
+}
+
+.firsttext {
+  font-family: "roboto", sans-serif;
+  font-size: 35px;
+  line-height: 72px;
+  text-align: center;
+  color: #7d73fc;
+  font-weight: 700;
+  letter-spacing: 1px;
+}
+
+.secondtext {
+  font-family: "roboto", sans-serif;
+  font-size: 35px;
+  line-height: 72px;
+  text-align: center;
+  color: white;
+  font-weight: 500;
+}
+
+.link {
+  text-decoration: none;
+  color: white;
+  align-items: center;
+  letter-spacing: 0.03em;
+  font-family: "roboto", sans-serif;
+  font-weight: 500;
+  font-size: 18px;
+  line-height: 16px;
+}
+
+p {
+  font-family: "roboto", sans-serif;
+  font-size: 20px;
+  text-align: center;
+  color: #c8c8d4;
+  font-weight: 400;
+  line-height: 28px;
+}
+
+body {
+  background-color: #191765;
+}
+
+i {
+  color: white;
+  fill: white;
```

Comment:
- [ ] It is recommended to use RGB or hexadecimal values for colors in CSS. Kindly fix that.